src/Typescriptr/Formatters/GeneratedCodeWarningCommentDecorator.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Typescriptr.Formatters
+{
+    public class GeneratedCodeWarningCommentDecorator
```

Comment:
And now that this doesn't have the warrning, lets name it `SourceLocationCommentDecorator` 🙂.